plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastore.java

```diff
@@ -754,6 +754,32 @@ public Map<String, Optional<Partition>> getPartitionsByNames(Table table, List<S
         }
     }
 
+    @Test
+    public void testDropTable()
```

Comment:
Thanks for the test.
Please make sure it fails on current master, without your fixes.  It is passing for me locally.

Reply:
@findepi Sorry i made a mistake, the bug looks like only exists in my internal modified version, because our community version has included the partition cache invalidation logic in `invalidateTable()`, like this:  
![image](https://user-images.githubusercontent.com/26461591/191201637-b7f0ebb1-b153-4211-ae64-cff7d3070db8.png)  
So it's not necessary to add `invalidatePartitionCache` in the try finally code block of `dropTable`. I'm sorry to bother you about the test, actually it should pass for you locally, wow it's a little embarrassing haha.  

But i still can contribute the unit test to our community, we lack the test about `dropTable` before, so this test can improve our existing tests.

Reply:
> But i still can contribute the unit test to our community, we lack the test about `dropTable` before, so this test can improve our existing tests.

Good idea, thanks